# Master joints limits
master_joint_min_x: -0.85    # elbow joint
master_joint_max_x: 1.25
master_joint_min_y: -1.0
master_joint_max_y: 1.0
master_joint_min_z: -1.0
master_joint_max_z: 1.0
master_joint_min_yaw: -2.57 # wrist3 joint
master_joint_max_yaw: 2.55
#master_joint_min_yaw: -1.0 # waist joint
#master_joint_max_yaw: 1.0

# Master cartesian limits (omni base frame)
master_min_x: 0.14
master_max_x: 0.22
master_min_y: -0.06
master_max_y: 0.06
master_min_z: -0.01
master_max_z: 0.20
master_min_yaw: -2.57 # wrist3 joint
master_max_yaw: 2.55

mode: 0
